dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -75,6 +113,8 @@ printf "%s\n" "Operating System Environment"
 printf "%s\n" "-----------------------------"
 printf "%25s: %s\n" "Operating System" "$(get_os_info)"
 printf "%25s: %s\n" "Kernel" "$(get_kernel_info)"
+printf "%25s: %s\n" "System shell" "$(get_sys_shell)"
+printf "%25s: %s\n" "make's shell" "$(get_make_shell)"
```

Comment:
I think the original was more accurate since `get_make_shell` is actually getting the value of makes `SHELL` variable. I would prefer it that way but wont insist.

```suggestion
printf "%25s: %s\n" "make's SHELL" "$(get_make_shell)"
```

Reply:
ok, makes sense - reverted and amended directly. Still works on macOS btw